assets/components/contributionPaymentCtas/contributionPaymentCtas.jsx

```diff
@@ -121,6 +121,7 @@ function OneOffCta(props: {
       url={clickUrl}
       onClick={onCtaClick(props.isDisabled, props.resetError)}
       id="qa-contribute-button"
+      modifierClasses={['contribute-one-off', 'border']}
```

Comment:
We have been calling this type of argument `modifiers` in [other parts of the codebase](https://github.com/guardian/support-frontend/search?q=modifiers&unscoped_q=modifiers), in order to be consistent we should probably keep it as `modifiers` or rename the other places to `modifierClasses`

Reply:
I would be more inclined to use `modifierClasses`, I'm not sure `modifiers` is descriptive enough at the call-site.

Reply:
Fair enough, in that case, is it possible to propagate the new name in the other places, so everything is consistent?